packages/extension/src/contentScript/utils/tests/getFrameType.ts

```diff
@@ -22,8 +22,28 @@ import '@testing-library/jest-dom';
  * Internal dependencies.
  */
 import getFrameType from '../getFrameType';
+import { I18n } from '@ps-analysis-tool/i18n';
```

Comment:
Some import from other package are under internal dependencies comment, please move them to appropriate place.